summerofcode/README.md

```diff
@@ -6,9 +6,10 @@ The Cloud Native Computing Foundation participates in the [Google Summer of Code
 
 If you need help with anything CNCF + Summer of code related, you can file an issue or contact one of the admins below:
 
-- Chris Aniszczyk ([@caniszczyk](https://github.com/caniszczyk)\): <https://twitter.com/cra>
-- Ihor Dvoretskyi ([@idvoretskyi](https://github.com/idvoretskyi)\): <https://twitter.com/idvoretskyi>
-- Nate Waddington ([@nate-double-u](https://github.com/nate-double-u)\): <https://twitter.com/hackabletype>
+- Chris Aniszczyk ([@caniszczyk](https://github.com/caniszczyk)) - Twitter: [@cra](https://twitter.com/cra), CNCF Slack: [@caniszczyk](https://cloud-native.slack.com/team/U0BB4K6KD)
+- Nate Waddington ([@nate-double-u](https://github.com/nate-double-u)) - Twitter: [@hackabletype](https://twitter.com/hackabletype), CNCF Slack: [@nate-double-u](https://cloud-native.slack.com/team/U01B9CR2A3E)
+- Jay Tiaki ([@jaytiaki](https://github.com/jaytiaki))
+- Ali Ok ([@aliok](https://github.com/aliok)) - CNCF Slack: [@Ali Ok](https://cloud-native.slack.com/team/U030UQH8QNR) 
```

Comment:
Notes/questions:
1. Do we want to keep Twitter here? Were you reached out by people over the last years over Twitter? If not, let's remove Twitter.
2. If we want to keep Twitter, @jaytiaki, would you like to provide me your Twitter handle, if you're using it and if you want it listed here.
3. I added direct links to Slack profiles. @jaytiaki I couldn't find you on CNCF Slack.

Reply:
Oh, and the links to Slack profiles are built using the doc [here](https://slack.com/help/articles/360003827751-Create-a-link-to-a-members-profile-) and the profile IDs are accessible anyway. Let me know if you find this a privacy/security concern.

Reply:
I'd prefer to drive folks to the cncf/mentoring discussion boards, rather than using individual contacts (and if we do want to link slack, I think it's probably better to link cncf #mentoring). I think that linking specific individuals this way sort of silos the information, and likely puts more work on the plate of the first person in the list. I'll let the others answer for themselves, but I'd prefer to drop my twitter link -- i've not had a lot of productive mentorship DMs there)

Historically we've had this list, but I'm not sure that we need it anymore. Thoughts from a historic perspective @idvoretskyi & @caniszczyk?

Reply:
Is there a GSoC requirement to list the admins outside of the platform?

Reply:
> Is there a GSoC requirement to list the admins outside of the platform?

I don't think so :)

> Historically we've had this list, but I'm not sure that we need it anymore. Thoughts from a historic perspective

I can remove this whole section and simply give instructions to reach out to #mentoring channel on Slack.